README.md

```diff
@@ -110,8 +110,8 @@ BlockScout supports a number of projects. Hosted instances include POA Network,
 
     ```shell
     cd docker
-    NETWORK=Celo ETHEREUM_JSONRPC_VARIANT=geth
-    ETHEREUM_JSONRPC_HTTP_URL=http://104.198.100.15:8545 ETHEREUM_JSONRPC_WS_URL=ws://104.198.100.15:8546 COIN=cGLD
+    export NETWORK=Celo ETHEREUM_JSONRPC_VARIANT=geth
```

Comment:
Actually, `export` shouldn't be needed in this case as that's the standard way of passing parameters to `make`.
```
make NETWORK=Celo ETHEREUM_JSONRPC_VARIANT=geth ETHEREUM_JSONRPC_HTTP_URL=http://104.198.100.15:8545 ETHEREUM_JSONRPC_WS_URL=ws://104.198.100.15:8546 COIN=cGLD start
```
should also work.